src/Microsoft.DncEng.Configuration.Extensions/ConfigurationConstants.cs

```diff
@@ -0,0 +1,10 @@
+namespace Microsoft.DncEng.Configuration.Extensions
+{
+    public static class ConfigurationConstants
```

Comment:
Why are these public (or constants)? Do we expect code outside of this assembly to be reaching into the configuration layer? That seems like a leaky abstraction.

Reply:
They are public just because I didn't make them private. They are the keys that need to be set for the configuration mapping to work. I will make them private.

Reply:
Actually, this needs to be public for now. The asp.net core data protection api `ProtectKeysWithAzureKeyVault` doesn't yet support the new azure sdks so the web sites need to use this constant to get the vault uri from the configuration.